activemodel/test/cases/validations/i18n_validation_test.rb

```diff
@@ -12,13 +12,17 @@ def setup
     I18n.load_path.clear
     I18n.backend = I18n::Backend::Simple.new
     I18n.backend.store_translations("en", errors: { messages: { custom: nil } })
+
+    @original_i18n_full_message = ActiveModel::Errors.i18n_full_message
+    ActiveModel::Errors.i18n_full_message = true
```

Comment:
Let's document this option in the configuring guide https://github.com/rails/rails/blob/master/guides/source/configuring.md.

Reply:
Added